Hazelcast.Test/Hazelcast.Client.Test.Serialization/PortableTest.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Threading.Tasks;
+using Hazelcast.IO.Serialization;
+using NUnit.Framework;
+
+namespace Hazelcast.Client.Test.Serialization
+{
+    internal class PortableTest
+    {
+        // Test for issue https://github.com/hazelcast/hazelcast/issues/12733
```

Comment:
we don't need this comment as this is .net fix. You can delete the comment

Reply:
Okay, It will be deleted in next commit